scattr/workflow/Snakefile

```diff
@@ -32,6 +32,15 @@ import os
 from pathlib import Path
 from functools import partial
 
+# Warnings
+if config.get("labelmerge_base_dir") or config.get("labelmerge_overlay_dir"):
```

Comment:
It's possible this more work than it's worth, but it would be nice to check if the metadata files are there and just error out if not. (Although maybe that will happen when labelmerge runs?)

Reply:
Yeah, I am (blindly) relying on labelmerge to perform this check since the `labelmerge_base_<opt>` flags are also used if one just wants to use their own custom atlas without any sort of merge.